catalog/geoscience-aus-cat/projects_geoscience-aus-cat_assets_ga_ls8cls9c_gm_cyear_3.jsonnet

```diff
@@ -161,8 +161,8 @@ local self_url = catalog_subdir_url + base_filename;
     nbart_swir_1: {minimum: 0, maximum: 10000, 'gee:estimated_range': true},
     nbart_swir_2: {minimum: 0, maximum: 10000, 'gee:estimated_range': true},
     edev: {minimum: 0, maximum: 10000, 'gee:estimated_range': true},
-    sdev: {minimum: 0, maximum: 10000, 'gee:estimated_range': true},
-    bcdev: {minimum: 0, maximum: 10000, 'gee:estimated_range': true},
+    sdev: {minimum: 0, maximum: 1, 'gee:estimated_range': true},
```

Comment:
This change is about updating another dataset, so ideally it should be in a separate PR.